frontend/web/components/pages/EnvironmentSettingsPage.js

```diff
@@ -690,8 +721,11 @@ const EnvironmentSettingsPage = class extends Component {
                           parentLevel='project'
                           parentSettingsLink={`/project/${this.props.match.params.projectId}/settings`}
                           id={this.props.match.params.environmentId}
+                          envId={env.id}
                           router={this.context.router}
                           level='environment'
+                          roleTabTitle='Environment Roles'
```

Comment:
> I think this is a bit misleading? It should rather be Role Environment permission? Because that's what it is, right? It is not a different role, but a permission attached to that role. Does that make sense?

Corrected.

>Also, do we need the text Can edit roles project permission. that written under that?

removed.